wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/internal/Util.kt

```diff
@@ -90,3 +96,55 @@ expect interface MutableQueue<T : Any> : MutableCollection<T> {
 }
 
 internal expect fun <T : Any> mutableQueueOf(): MutableQueue<T>
+
+// TODO internal and friend for wire-compiler: https://youtrack.jetbrains.com/issue/KT-20760
```

Comment:
The ticket is fixed in 1.3.60?

Reply:
I believe the compiler is fixed but the IDE is not. I just copied the issue link from above. The IDE issue is https://youtrack.jetbrains.com/issue/KT-34102. I can update all the links separately.